### ISSUE TITLE
[1.19.2] Cursemaven

### ISSUE THREAD

**Issue by Christofmeg** [TinyModularThings/IC2Classic#610]
Trying to runData or runClient when project is built against latest file on cursemaven (1.19.2-2.0.3 https://www.curseforge.com/minecraft/mc-mods/ic2-classic/files/4348706) results in this:

```
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:68) ~[javafmllanguage-1.19.2-43.2.0.jar%23184!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:121) ~[fmlcore-1.19.2-43.2.0.jar%23187!/:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?] {}
Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.client.Minecraft.getResourcePackRepository()" because the return value of "net.minecraft.client.Minecraft.getInstance()" is null
	at ic2.core.platform.rendering.PackHack.doResourcePackHack(PackHack.java:48) ~[ic2_classic-242942-4348706_mapped_parchment_2022.11.27-1.19.2.jar%23200!/:1.19.2-2.0.3] {re:classloading,pl:runtimedistcleaner:A}
	at ic2.core.PlatformClient.preInit(PlatformClient.java:144) ~[ic2_classic-242942-4348706_mapped_parchment_2022.11.27-1.19.2.jar%23200!/:1.19.2-2.0.3] {re:classloading,pl:runtimedistcleaner:A}
	at ic2.core.IC2.<init>(IC2.java:160) ~[ic2_classic-242942-4348706_mapped_parchment_2022.11.27-1.19.2.jar%23200!/:1.19.2-2.0.3] {re:classloading}
	... 14 more
```

Using forge 1.19.2-43.2.0

From build.gradle:
```
repositories {
    maven {
        url "https://www.cursemaven.com"
            content {
                includeGroup "curse.maven"
            }
        }
}
dependencies {
    minecraft 'net.minecraftforge:forge:1.19.2-43.2.0'
    implementation fg.deobf("curse.maven:ic2_classic-242942:4348706")
}
```

**Comment by Speiger** [comment]
Not sure what the issue is.
But i know addons like gravity suit work fine with curse maven.
Here is their gradle script.
https://github.com/Trins-mods/Gravisuit-Classic-Edition/blob/master/build.gradle

**Comment by Speiger** [comment]
@Trinsdar / @Meduris please provide input here.

**Comment by Meduris** [comment]
the only strange thing I noticed so far are the mixin related properties.

I never had issues with including IC2 Classic as a dependency, though I am using custom local builds instead of the versions published to curseforge. Also I never used the data configuration so far

**Comment by Christofmeg** [comment]
Well, removing mixins from data runs:

```
//            property 'mixin.env.remapRefMap', 'true'
//            property 'mixin.env.refMapRemappingFile', "${projectDir}/build/createSrgToMcp/output.srg"
```

Makes the log point towards IC2 BlockMixin

```
[08:19:59] [main/DEBUG] [mixin/]: Mixing server.BlockMixin from ic2.mixins.json into net.minecraft.world.level.block.state.BlockBehaviour
[08:19:59] [main/FATAL] [mixin/]: Mixin apply failed ic2.mixins.json:server.BlockMixin -> net.minecraft.world.level.block.state.BlockBehaviour: org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException No candidates were found matching f_60439_:Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; in net/minecraft/world/level/block/state/BlockBehaviour for ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; [INJECT Applicator Phase -> ic2.mixins.json:server.BlockMixin -> Apply Accessors ->  -> Locate -> ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties;]
org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException: No candidates were found matching f_60439_:Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; in net/minecraft/world/level/block/state/BlockBehaviour for ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; [INJECT Applicator Phase -> ic2.mixins.json:server.BlockMixin -> Apply Accessors ->  -> Locate -> ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties;]
	at org.spongepowered.asm.mixin.gen.AccessorInfo.findTarget(AccessorInfo.java:518) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.gen.AccessorInfo.findTargetField(AccessorInfo.java:501) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.gen.AccessorInfo.locate(AccessorInfo.java:475) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.generateAccessors(MixinTargetContext.java:1389) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyAccessors(MixinApplicatorStandard.java:1060) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:398) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.8.jar:10.0.8+10.0.8+main.0ef7e830] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.4.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?] {}
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:119) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.4.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.4.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
	at net.minecraftforge.registries.GameData.init(GameData.java:110) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23182%23189!/:?] {re:classloading}
	at net.minecraftforge.registries.GameData.<clinit>(GameData.java:95) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23182%23189!/:?] {re:classloading}
	at net.minecraft.core.Registry.forge(Registry.java:438) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23183!/:?] {re:classloading}
	at net.minecraft.core.Registry.forge(Registry.java:422) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23183!/:?] {re:classloading}
	at net.minecraft.core.Registry.<clinit>(Registry.java:186) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23183!/:?] {re:classloading}
	at net.minecraft.server.Bootstrap.bootStrap(Bootstrap.java:47) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23183!/:?] {re:classloading}
	at net.minecraftforge.data.loading.DatagenModLoader.begin(DatagenModLoader.java:35) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23182%23189!/:?] {re:classloading}
	at net.minecraft.data.Main.main(Main.java:76) ~[forge-1.19.2-43.2.0_mapped_parchment_2022.11.27-1.19.2-recomp.jar%23183!/:?] {re:classloading}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.ForgeDataUserdevLaunchHandler.lambda$launchService$0(ForgeDataUserdevLaunchHandler.java:28) ~[fmlloader-1.19.2-43.2.0.jar%2395!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%23108!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?] {}
Exception in thread "main" java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:32)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.19.2-43.2.0/net.minecraftforge.fml.loading.targets.ForgeDataUserdevLaunchHandler.lambda$launchService$0(ForgeDataUserdevLaunchHandler.java:28)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
	... 7 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:119)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219)
	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at TRANSFORMER/forge@43.2.0/net.minecraftforge.registries.GameData.init(GameData.java:110)
	at TRANSFORMER/forge@43.2.0/net.minecraftforge.registries.GameData.<clinit>(GameData.java:95)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.core.Registry.forge(Registry.java:438)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.core.Registry.forge(Registry.java:422)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.core.Registry.<clinit>(Registry.java:186)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.server.Bootstrap.bootStrap(Bootstrap.java:47)
	at TRANSFORMER/forge@43.2.0/net.minecraftforge.data.loading.DatagenModLoader.begin(DatagenModLoader.java:35)
	at TRANSFORMER/minecraft@1.19.2/net.minecraft.data.Main.main(Main.java:76)
	... 13 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [ic2.mixins.json:server.BlockMixin] from phase [DEFAULT] in config [ic2.mixins.json] FAILED during APPLY
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:636)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:588)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379)
	... 42 more
Caused by: org.spongepowered.asm.mixin.gen.throwables.InvalidAccessorException: No candidates were found matching f_60439_:Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; in net/minecraft/world/level/block/state/BlockBehaviour for ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties; [INJECT Applicator Phase -> ic2.mixins.json:server.BlockMixin -> Apply Accessors ->  -> Locate -> ic2.mixins.json:server.BlockMixin->@Accessor[FIELD_GETTER]::getProperties()Lnet/minecraft/world/level/block/state/BlockBehaviour$Properties;]
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.gen.AccessorInfo.findTarget(AccessorInfo.java:518)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.gen.AccessorInfo.findTargetField(AccessorInfo.java:501)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.gen.AccessorInfo.locate(AccessorInfo.java:475)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinTargetContext.generateAccessors(MixinTargetContext.java:1389)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyAccessors(MixinApplicatorStandard.java:1060)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:398)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365)
	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 42 more
```

**Comment by Speiger** [comment]
I am sorry but I can not help you with that.
What we did was simply this with lithium for example:

```groovy
dependencies {
	implementation fg.deobf("curse.maven:rubidium-574856:3973894") 
}
```

Thats all we used and it worked.
We do nothing special with mixins in fact anything that could have been special was removed after the beta concluded.

I am sorry, but i am not capable of helping you.

**Comment by Meduris** [comment]
small update on this. from what I found online, these two lines are absolutely necessary when you add mods with mixins as dependencies:
```
property 'mixin.env.remapRefMap', 'true'
property 'mixin.env.refMapRemappingFile', "${projectDir}/build/createSrgToMcp/output.srg"
```

sorry for the confusion about that. I really didn't know about that part as my setup is a bit special and didn't need that to work
(found an issue at immersive engineering about that topic, especially these two comments: [comment 1](https://github.com/BluSunrize/ImmersiveEngineering/issues/5178#issuecomment-1011211404) and [comment 2](https://github.com/BluSunrize/ImmersiveEngineering/issues/5178#issuecomment-1011287352))

for the runData configuration, I did some further digging and it looks like this is our fault as we have multiple places, where we just assume that we have a running minecraft instance and (client side) a minecraft window.

However from what I saw, these two assumptions are completely wrong when running the runData configuration.
